src/main/java/com/gbtec/gameoflife/custom/GameOfLifeService.java

```diff
@@ -0,0 +1,47 @@
+package com.gbtec.gameoflife.custom;
```

Comment:
"service" as a package name would not fit, because i thought, that the implementer can place all his classes inside the package (including subpackages)

Reply:
Then I would recommend to rename into the following:
basic -> framework
custom -> implementation